packages/web/src/components/track-availability-modal/TrackAvailabilityModal.tsx

```diff
@@ -71,6 +71,7 @@ const LEARN_MORE_URL =
 const defaultAvailabilityFields = {
   is_premium: false,
   premium_conditions: null,
+  preview_start_seconds: null,
```

Comment:
I might be confused, but isn't this whole modal getting replaced with the `AccessAndSaleModalLegacy`? Shouldn't we just delete this whole file? Or is this still being used elsewhere (like edit modal?)

Reply:
Ah yeah I think we're likely still using this for edit modal since we're not accounting for the non-upload flow disabled states of radio buttons in the new modal fields?

Reply:
Thanks for the context on edit. I'm planning to hold this PR back for tonight's release cut as the change to use the new modal is not behind any feature flag. That'll give me time to through the edit flows and make sure it's behaving properly there.